packages/react-draggable-dialog/src/hooks/useDraggableDialog.ts

```diff
@@ -0,0 +1,5 @@
+import { useDraggableDialogContext } from '../contexts/DraggableDialogContext';
```

Comment:
thought: any reason why this i not part of contexts ? this seems unnecessary separation by wrong type of entity, while from domain POV this should live with context.

Reply:
More logic was meant to be added, but for simplification, I reverted this change. It now lives with the context.